.github/workflows/publish-commit-bottles.yml

```diff
@@ -103,7 +104,13 @@ jobs:
             exit 1
           fi
 
-          echo "branch=$branch" >> "$GITHUB_OUTPUT"
+          if [ "$branch" = "master" ]
+          then
+            echo "branch=$head_repo_owner/$branch" >> "$GITHUB_OUTPUT"
+          else
+            echo "branch=$branch" >> "$GITHUB_OUTPUT"
+          fi
+          echo "origin_branch=$branch" >> "$GITHUB_OUTPUT"
           echo "remote=$remote" >> "$GITHUB_OUTPUT"
 
           if "$pushable" && [ "$fork_type" != "Organization" ]
```

Comment:
While we're here we should fix this check too. This prevents merging PRs from branches in this repository. 🙃

Reply:
Oh, okay... 😅

Fixed in 1333ef90f227a466709134c3ee1c237bd5fe1b3b.